<launch>
  <arg name="diag_name" />

  <group ns="dummy_diag_publisher">
    <node pkg="dummy_diag_publisher" exec="dummy_diag_publisher" name="$(var diag_name)" output="screen">
      <param name="use_sim_time" value="$(env AW_ROS2_USE_SIM_TIME false)"/>
      <param from="$(find-pkg-share dummy_diag_publisher)/config/$(var diag_name).param.yaml" />
    </node>
  </group>
</launch>
